wallet/src/main/java/com/android/identity_credential/wallet/WalletApplication.kt

```diff
@@ -62,5 +71,26 @@ class WalletApplication : Application() {
         issuingAuthorityRepository = IssuingAuthorityRepository()
         issuingAuthorityRepository.add(SelfCertificationIssuingAuthority(this, storageEngine))
         issuingAuthorityRepository.add(PassportBasedIssuingAuthority(this, storageEngine))
+
+        initTrustManager()
+    }
+
+    private fun initTrustManager(){
+        // init Trust Manager
+        // read TrustPoint owf pem cert to manually add an "owf identity credential reader" (appverifier)
```

Comment:
You're _kinda_ stating the obvious in both these comment lines, I think those can be deleted.